docs/developer-guide/rules.md

```diff
@@ -1,57 +1,81 @@
-# Writing rules
+# Working on rules
+
+**Please help us create, enhance, and debug stylelint rules!**
+
+There are well over a hundred rules already, so stylelint *needs* community contributions to continue its growth.
```

Comment:
Replace "continue its growth", with something like "stay in shape" (for the lack of a better expression)?